File: Java/OddEven.java
/*
WAP to check if the input number is Odd or even


Contact: mailtopksubudhi@gmail.com
WhatsApp: +91-8895174939
website: www.pksubudhi.com
*/

import java.lang.*;
import java.util.*;
class OddEven
{
	public static void main (String args[])
	{
		int x;
		Scanner inp=new Scanner(System.in);
		System.out.print("Type an integer: ");
		x=inp.nextInt();
		if(x%2==0)
		{
			System.out.println("Even");
		}
		else
		{
			System.out.println("Odd");
		}
	}
}

File: Python/OddEven.py
# To read a number from console we have to use input() function
# However, it returns the input value as a string
# To parse a string into integer, we can use int() function

x = int(input("Type an Integer: "))
if (x % 2) == 0:
    print(x, "is an Even Number")
else:
    print(x, "is an Odd Number")
